core/includes/drupal.inc

```diff
@@ -2591,6 +2591,51 @@ function views_clean_css_identifier($identifier, $filter = array(' ' => '-', '/'
   backdrop_clean_css_identifier($identifier, $filter);
 }
 
+/**
+ * Finds the most recently changed nodes that are available to the current user.
+ *
+ * @param (int) $number
```

Comment:
The variable name needs to be updated in the docblock as well.
```suggestion
 * @param (int) $limit
```

Reply:
Done